docs/UserGuide.md

```diff
@@ -304,9 +304,22 @@ Shows a list of all tasks in the task list.
 <div markdown="block" class="alert alert-primary">
 
 **:clipboard: Format:**<br>
-- `list`
+- <code> `list` </code>
+
+</div>
+
+<div markdown="block" class="alert alert-success">
+
+**:green_book: Example:**
+
+* `list` displays a list of all the task in the task list
+
 </div>
 
+After running the list command, the tasks in that given week are listed as shown below:
+
+![find_command_after](images/listCommand.png)
```

Comment:
Could edit the title of image "find_command_after" to "list_command_after"

Reply:
Noted